tensorboard/uploader/uploader_test.py

```diff
@@ -72,6 +72,7 @@ def _create_mock_client():
 
 _SCALARS_ONLY = frozenset((scalars_metadata.PLUGIN_NAME,))
 
+_TEN_MEGABYTES = 10 * (2 ** 20)
```

Comment:
This kind of `const TWELVE = 12;` definition is generally considered an
antipattern; if we want a constant here, can it describe the semantics
rather than the value? Also fine with me to just inline it, as it looks
like the particular value is not important (which is good, imho).

Reply:
Yeah I totally agree that's an antipattern.  I must have been quite addled at the time.  I inlined `12345` to emphasize that the value is arbitrary here.

Reply:
:-) thanks; sgtm!